tesseract/tesseract_planning/tesseract_process_managers/src/process_managers/raster_dt_process_manager.cpp

```diff
@@ -201,6 +207,14 @@ bool RasterDTProcessManager::execute()
 {
   success_ = true;
 
+  DebugObserver::Ptr debug_observer;
+  std::shared_ptr<tf::TFProfObserver> profile_observer;
+  if (debug_)
+    debug_observer = executor_.make_observer<DebugObserver>("RasterDTProcessManagerObserver");
+
+  if (profile_)
+    profile_observer = executor_.make_observer<tf::TFProfObserver>();
```

Comment:
Updated section along with saving it to a file below.

Reply:
I am also updating in the Planning Request on the ROS side

Reply:
Oh ok. I didn't realize it was in Taskflow. I'll have to try it sometime

Reply:
Yea neither did I until recently. I ask a question looking for a way to trace the path taken through the graph and they pointed me to this.